moped-database/migrations/1696624355279_add-length-to-views/up.sql

```diff
@@ -0,0 +1,95 @@
+CREATE OR REPLACE VIEW "public"."uniform_features" AS
```

Comment:
we do not have to drop the view bc the schema is not changing, because the new length column is tucked into the existing `attributes` 😅